_posts/members-meetings/2020-06-13-agm-minutes.md

```diff
@@ -0,0 +1,836 @@
+# Hypha 2020 AGM - June 13-14, 2020
+
+_These are draft minutes from our initial AGM._
```

Comment:
```suggestion
_These are minutes from our inaugural AGM._
```

Reply:
If they haven't been approved we need to indicate that somewhere